js/src/Ice/LocalException.js

```diff
@@ -5,12 +5,7 @@
 import { Exception } from "./Exception.js";
 
 export class LocalException extends Exception {
-    constructor(cause) {
-        super(cause);
-        Exception.captureStackTrace(this);
-    }
-
-    static get _id() {
-        return "::Ice::LocalException";
+    constructor(message, cause) {
```

Comment:
I lean toward simply inheriting the constructor. In particular, when you throw a derived local exception with a cause, we should use the JS error syntax:
```js
throw new MarshalException("Message", { cause: err });
```

Providing another syntax sounds confusing.